pages/about.js

```diff
@@ -0,0 +1,97 @@
+import Headline from "@/components/Headline";
+import Image from "next/image";
+import styled from "styled-components";
+import Link from "next/link";
+import { StyledTitle } from "@/components/Title/StyledTitle";
+
+export default function AboutPage() {
+  return (
+    <>
+  <Headline />
+  <StyledTitle>About Us</StyledTitle>
+
+  <>
+    <StyledAboutContainer>
+      <StyledAbout>
+        <StyledEntryCard>
+          <Image src="/images/Jan.png" width={100} height={100} alt="Uploaded" />
+          <div>
+            <StyledFigcaption>Jan</StyledFigcaption>
+            <Link href="https://github.com/jantillidie">Github</Link>
+          </div>
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure), this should probably look like this
```suggestion
       <figure>
          <Image src="/images/Jan.png" width={100} height={100} alt="Uploaded" />
          <StyledFigcaption>Jan</StyledFigcaption>
       </figure>
       <Link href="https://github.com/jantillidie">Github</Link>
```

I'm not sure if `figure` gets along with next/image, but usually you'd nest the ìmg` and `figcaption` elements inside of a `figure` element. 🤔